heltour/tournament/migrations/0192_player_account_status.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.13 on 2021-10-08 05:40
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    
+    def update_player_status(apps, schema_editor):
+        Players = apps.get_model('tournament', 'Player')
+        for player in Players.objects.all():
```

Comment:
This is better written:
`Players.objects.filter(account_status__in=('engine', 'booster')).update(account_status='tos_violation')`

as this will be a single query.

Reply:
done